rest/changestest/changes_api_test.go

```diff
@@ -3979,8 +3985,12 @@ func WriteDirectWithKey(testDb *db.DatabaseContext, key string, channelArray []s
 	syncData.TimeSaved = time.Now()
 	// syncData := fmt.Sprintf(`{"rev":"%s", "sequence":%d, "channels":%s, "TimeSaved":"%s"}`, rev, sequence, chanMap, time.Now())
 
-	_, err := testDb.Bucket.DefaultDataStore().Add(key, 0, db.Body{base.SyncPropertyName: syncData, "key": key})
+	dataStores, _ := testDb.Bucket.ListDataStores()
```

Comment:
I think collection.dataStore is the data store you're looking for here.  I think it's reasonable to add a public accessor to make that available, so that WriteDirect can just take DatabaseCollection instead of DatabaseContext.